### PR TITLE
Adding ACMESharp and tug projects

### PR DESCRIPTION
**Pull request by ebekker**

### CODE REVIEW

File: _data/projects/ACMESharp.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Note those are tags categorizing your project, not issue labels.
+- ".NET"
+- C#
+- PowerShell
+- Security
+- AWS
+- IIS
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/ebekker/ACMESharp/issues?q=is%3Aopen+is%3Aissue+label%3Aup-for-grabs
```
Comment (shiftkey) [inline _data/projects/ACMESharp.yml]: :lipstick: you can simplify this down to `https://github.com/ebekker/ACMESharp/labels/up-for-grabs`

**Comment by shiftkey** [comment]
@ebekker looks good, just that one little improvement

**Comment by ebekker** [comment]
@shiftkey -- sorry, your comment slipped by me a while back and never got around to it till just now!

**Comment by shiftkey** [comment]
@ebekker all good - thanks!

**Pull request merged**